packages/babel-preset-env/data/plugins.json

```diff
@@ -299,6 +299,10 @@
     "opera": "53",
     "electron": "3.1"
   },
+  "proposal-optional-chaining": {
```

Comment:
We should also add a plugin syntax map to the `pluginSyntaxMap`
https://github.com/babel/babel/blob/bb6cc619796666668c8228094bf6beecfbb86ce4/packages/babel-preset-env/data/shipped-proposals.js#L6

So that if babel is configured with
```json
{
  "presets": [
     ["@babel/env", { "targets": { "browserslist": "chrome 80" } }]
   ]
}
```
Babel would _not_ transform them at all since it gets native support.

By doing so it means we need to add `@babel/syntax-proposal-optional-chaining` as dependencies, too.

Reply:
Added - hopefully I did that correctly. I must admit, I don't fully understand all of the machinery here, so assume the worst 😂 

Thanks for the quick feedback, as well 👍

Reply:
Hmm, am I correct in assuming I should also add the syntax plugin to the `notIncludedPlugins` in `babel-present-env-standalone`?

https://github.com/babel/babel/blob/bb6cc619796666668c8228094bf6beecfbb86ce4/packages/babel-preset-env-standalone/src/available-plugins.js#L3

Reply:
You are doing great! And we should also add the syntax plugin to preset-env/src/available-plugins.